packages/eslint-config-airbnb-base/index.js

```diff
@@ -0,0 +1,13 @@
+module.exports = {
+  extends: [
+    './legacy',
+    './rules/es6',
+  ].map(require.resolve),
+  parserOptions: {
+    ecmaVersion: 7,
+    sourceType: 'module',
+  },
+  rules: {
+    strict: 2,
```

Comment:
I added a number of new rules, such that it's not simply the same as shuffling dependencies.

Why do you need ecmaVersion 6 instead of 7? The only difference is the exponentiation operator.

At any rate, our configs mirror what we use internally at Airbnb, and we transpile up to ES7.